infra/gcp/lib_gcs.sh

```diff
@@ -56,6 +56,28 @@ function ensure_public_gcs_bucket() {
     ensure_gcs_role_binding "${bucket}" "allUsers" "objectViewer"
 }
 
+# Set up logging
+# $1: The GCS bucket
+function ensure_gcs_bucket_logging() {
+    if [ $# != 2 ] || [ -z "$1" ]; then
+        echo "ensure_gcs_bucket_logging(name) requires 1 argument" >&2
+        return 1
+    fi
+    local bucket="$1"
```

Comment:
For parity with other functions in this library, give an example, and stick with bucket vs. bucket name
```suggestion
# $1: The GCS bucket (e.g. gs://k8s-infra-foo)
function ensure_gcs_bucket_logging() {
    if [ $# != 2 ] || [ -z "$1" ]; then
        echo "ensure_gcs_bucket_logging(bucket) requires 1 argument" >&2
        return 1
    fi
    local bucket="$1"
```

Reply:
whoops, one more... it should be `[ $# != 1 ]`